tutorials/sapui5-fiori-cf-display-data/sapui5-fiori-cf-display-data.md

```diff
@@ -39,25 +39,34 @@ The routes are added to the `uimodule/webapp/manifest.json` file. The generator
 [ACCORDION-END]
 [ACCORDION-BEGIN [Step : ](Inspect the modifications)]
 
-As you can see in the log, there are two new files and one modified file. As the generator is only able to create boilerplate code, we have to make some modifications to the `uimodule/webapp/manifest.json` application descriptor.
-
-**Open** the file and replace the routing pattern of the new view with an empty string.
-```JSON [3]
-{
-  "name": "Products",
-  "pattern": "",
-  "target": [
-    "TargetProducts"
-  ]
-},
-```
+1. As you can see in the log, there are two new files and one modified file. As the generator is only able to create boilerplate code, we have to make some modifications to the `uimodule/webapp/manifest.json` application descriptor.
```

Comment:
Please don't say "we" when addressing the user, say "you" instead.

Reply:
done, thanks a lot